src/core/common/notifier.hpp

```diff
@@ -66,6 +66,13 @@ namespace ot {
 class Notifier : public InstanceLocator
 {
 public:
+    /**
+     * This type defines a bit-field indicating specific state or configuration that has changed. See
+     * `OT_CHANGED_<STATE>` definitions in `instance.h`.
+     *
+     */
+    typedef otChangedFlags Flags;
```

Comment:
I'm not sure what the advantage of this typedef is?  It would seem more readable to just use otChangedFlags throughout?

Reply:
Fully agree. Changed this in new pushed commit.